crates/starknet/src/messaging/ethereum.rs

```diff
@@ -371,11 +318,13 @@ impl EthereumMessaging {
                 e
             )))
         })?
-        .send()
-        .await
-        .map_err(|e| {
-            Error::MessagingError(MessagingError::EthersError(format!(
-                "Error deploying messaging contract: {}",
+            // Required by the new version of anvil, as default is no longer accepted.
+            .gas_price(1000000000)
```

Comment:
How did you choose this value? How likely is it to fail?

Reply:
I took the exact value that is used by anvil and hardhat as base fee.
I can put a bit more just to ensure that if this default value changes in the future it will not break.

Reply:
Comment added in 9cc2be46c51128155506d036c1d53621f281e79d and I multiplied by 20 to have a safe value as the default value is not expected to change everyday by anvil/hardhat releases.